android/build.gradle

```diff
@@ -1,13 +1,23 @@
 // Top-level build file where you can add configuration options common to all sub-projects/modules.
 
+def REACT_NATIVE_VERSION = new File(['node', '--print',"JSON.parse(require('fs').readFileSync(require.resolve('react-native/package.json'), 'utf-8')).version"].execute(null, rootDir).text.trim())
+
 buildscript {
     ext {
         minSdkVersion = 23
-        compileSdkVersion = 30
-        targetSdkVersion = 30
+        compileSdkVersion = 31
+        targetSdkVersion = 31
         supportLibVersion = "30.0.0"
         RNNKotlinVersion = "1.3.61" // Or any version above 1.3.x
         RNNKotlinStdlib = "kotlin-stdlib-jdk8"
+
+        if (System.properties['os.arch'] == "aarch64") {
+            // For M1 Users we need to use the NDK 24 which added support for aarch64
+            ndkVersion = "24.0.8215888"
```

Comment:
Is there a reason not to use this ``ndkVersion``  as the minimum for all platforms?